packages/cf-component-checkbox/test/Checkbox.js

```diff
@@ -1,8 +1,44 @@
 const React = require('react');
 const assertEqualJSX = require('assert-equal-jsx');
 const Checkbox = require('../src/Checkbox');
+const {expect} = require('chai');
+const {render, unmountComponentAtNode} = require('react-dom');
+const TestUtils = require('react-addons-test-utils');
+
+var then = function (callback, timeout) {
+  setTimeout(callback, timeout > 0 ? timeout : 0);
+  return {then: then};
+};
```

Comment:
I think we are using enzyme in a couple of places already. Enzyme's simulate will grab the onClick handler and call it directly. This may make the tests more straightforward conceptually.

http://airbnb.io/enzyme/docs/api/ShallowWrapper/simulate.html